src/runner/browser-job.ts

```diff
@@ -214,13 +214,20 @@ export default class BrowserJob extends AsyncEventEmitter {
         return this._testRunControllerQueue;
     }
 
+    private _getlastIndexFixture ( fixtureId: string ): number {
+        const lastIndexFixture = this._testRunControllerQueue.findIndex(el => el.test.fixture?.id !== fixtureId);
```

Comment:
What do you mean by `lastIndexFixture`?

Reply:
Last index current fixture with disable concurrency

Reply:
I rename `lastIndexFixture` and method ` _getlastIndexFixture` to  `nextFixtureFirstTestIndex` and `_getNextFixtureFirstTestIndex`. Last names was incorrect.

Method returns next fixture's test's first index.